packages/foreman/foreman/foreman.spec

```diff
@@ -251,79 +251,80 @@ BuildRequires: %{?scl_prefix}rubygem(graphql-batch)
 BuildRequires: %{scl}-runtime-assets < 6
+%else
+BuildRequires: nodejs-packaging
 %endif
-BuildRequires: nodejs >= 6.10
+BuildRequires: %{?scl_prefix_nodejs}nodejs >= 6.10
 BuildRequires: http-parser
 # Temporary dep on libuv until https://bugs.centos.org/view.php?id=10606
 # is resolved
 BuildRequires: libuv
-BuildRequires: nodejs-packaging
 BuildRequires: systemd
 
 # start package.json devDependencies BuildRequires
-BuildRequires: npm(argv-parse) >= 1.0.1
-BuildRequires: npm(argv-parse) < 2.0.0
-BuildRequires: npm(babel-cli) >= 6.10.1
-BuildRequires: npm(babel-cli) < 7.0.0
-BuildRequires: npm(babel-core) >= 6.26.3
-BuildRequires: npm(babel-core) < 7.0.0
-BuildRequires: npm(babel-loader) >= 7.1.1
-BuildRequires: npm(babel-loader) < 8.0.0
-BuildRequires: npm(babel-plugin-syntax-dynamic-import) >= 6.18.0
-BuildRequires: npm(babel-plugin-syntax-dynamic-import) < 7.0.0
-BuildRequires: npm(babel-plugin-transform-class-properties) >= 6.24.1
-BuildRequires: npm(babel-plugin-transform-class-properties) < 7.0.0
-BuildRequires: npm(babel-plugin-transform-object-assign) >= 6.8.0
-BuildRequires: npm(babel-plugin-transform-object-assign) < 7.0.0
-BuildRequires: npm(babel-plugin-transform-object-rest-spread) >= 6.8.0
-BuildRequires: npm(babel-plugin-transform-object-rest-spread) < 7.0.0
-BuildRequires: npm(babel-preset-env) >= 1.7.0
-BuildRequires: npm(babel-preset-env) < 2.0.0
-BuildRequires: npm(babel-preset-react) >= 6.5.0
-BuildRequires: npm(babel-preset-react) < 7.0.0
-BuildRequires: npm(babel-register) >= 6.9.0
-BuildRequires: npm(babel-register) < 7.0.0
-BuildRequires: npm(compression-webpack-plugin) >= 1.1.11
-BuildRequires: npm(compression-webpack-plugin) < 1.2.0
-BuildRequires: npm(css-loader) >= 0.23.1
-BuildRequires: npm(css-loader) < 1.0.0
-BuildRequires: npm(dotenv) >= 5.0.0
-BuildRequires: npm(dotenv) < 6.0.0
-BuildRequires: npm(expose-loader) >= 0.6.0
-BuildRequires: npm(expose-loader) < 0.7.0
-BuildRequires: npm(extract-text-webpack-plugin) >= 3.0.0
-BuildRequires: npm(extract-text-webpack-plugin) < 4.0.0
-BuildRequires: npm(file-loader) >= 0.9.0
-BuildRequires: npm(file-loader) < 1.0.0
-BuildRequires: npm(identity-obj-proxy) >= 3.0.0
-BuildRequires: npm(identity-obj-proxy) < 4.0.0
-BuildRequires: npm(node-sass) >= 4.5.0
-BuildRequires: npm(node-sass) < 5.0.0
-BuildRequires: npm(raf) >= 3.4.0
-BuildRequires: npm(raf) < 4.0.0
-BuildRequires: npm(sass-loader) >= 6.0.6
-BuildRequires: npm(sass-loader) < 6.1.0
-BuildRequires: npm(style-loader) >= 0.13.1
-BuildRequires: npm(style-loader) < 1.0.0
-BuildRequires: npm(uglifyjs-webpack-plugin) >= 1.2.2
-BuildRequires: npm(uglifyjs-webpack-plugin) < 2.0.0
-BuildRequires: npm(url-loader) >= 1.0.1
-BuildRequires: npm(url-loader) < 2.0.0
-BuildRequires: npm(webpack) >= 3.4.1
-BuildRequires: npm(webpack) < 4.0.0
-BuildRequires: npm(webpack-stats-plugin) >= 0.1.5
-BuildRequires: npm(webpack-stats-plugin) < 1.0.0
+BuildRequires: %{?scl_prefix}npm(argv-parse) >= 1.0.1
```

Comment:
should all these be `scl_prefix_nodejs`?

Reply:
I think these are packages from our own repo so I think it's correct. However, we haven't built these yet in our koji.

Reply:
makes sense.